charts/fluent-bit/values.yaml

```diff
@@ -31,6 +31,7 @@ serviceAccount:
   create: true
   annotations: {}
   name:
+  automountServiceAccountToken: true
```

Comment:
```suggestion
  automountServiceAccountToken:
```

To align with the suggestion above this would be the default value to not set the value in the spec.